core/src/main/resources/lib/layout/html.jelly

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
For maintainability it is desired to avoid substantial duplication of less-than-obvious code.